src/components/categoryPage.jsx

```diff
@@ -3,17 +3,20 @@ import axios from "axios";
 import { useParams } from "react-router-dom";
 
 import ReviewCard from "./reviewCard";
+import Loading from "./loading";
 export default function Category(){
     const {slug} = useParams()
     const [reviewByCategory, setReviewByCategory] = useState([])
+    const [isLoading, setIsLoading] = useState(true)
     useEffect(()=>{
```

Comment:
dont forget to setIsLoading to true here! when this function gets called again you want to set loading to true before fetching the info, so there isn't a lag between the screen changing and having the old reviews on there :)